src/styles/blocks/card/card.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 200px;
```

Comment:
move `width` to `catalog__card` as well. it's an external style
<img width="913" alt="image" src="https://user-images.githubusercontent.com/102411130/209311402-db2a446c-86ac-43a5-885d-8795f3aad654.png">